neo4j/src/main/java/org/hibernate/ogm/datastore/neo4j/transaction/impl/Neo4jLocalTransactionCoordinator.java

```diff
@@ -20,7 +20,7 @@
  *
  * Note that during a JTA transaction Neo4j {@link Transaction} are
  * synchronized using the {@link Synchronization} interface. A commit to the Neo4j transaction will happen before the
- * end of the JTA transaction, meaning that it won't be possible to rollback if an error happen after succesful commit
```

Comment:
I haven't figured yet how these pieces fit together. How does JTA play in here, given this is a "local transaction coordinator"?